Wyjątki dzielimy na kontrolowane które trzeba obsłużyć i niekontrolowane które można, ale nie trzeba obsłużyć.

Wyjątki exception dziedziczą po klasie RuntimeException

Wyjątki rzucamy korzystając z instrukcji throw new Exception(), jeśli rzucamy wyjątek kontrolowany musimy dodatkowo w sygnaturze metody/konstruktora zadeklarować wyjątek używając słowa throws Exception

Do bloku try-catch możemy dodać dodatkowo blok finally, który wykona się zawsze, niezależnie, czy w bloku try wyjątek wystąpił, czy nie